dae/dae/variants_loaders/cnv/tests/test_cnv_loader.py

```diff
@@ -133,7 +135,7 @@ def test_cnv_loader_alt_2(fixture_dirname, gpf_instance_2013):
     )
 
     svs = []
-    fvs = []
+    fvs: List[Any] = []
```

Comment:
This is not a criticism, just a suggestion - maybe we should use `list[..]` instead of `List[...]`.

Reply:
Agreed. Is it possible to enforce this convention using mypy or pylint?